src/cdomains/affineEquality/convenienceOps.ml

```diff
@@ -0,0 +1,18 @@
+open RatOps
+
+(** It provides more readable infix operators for the functions of RatOps.
+    It is designed to be included by modules that make use of RatOps's functions. *)
+module ConvenienceOps (A: RatOps) =
```

Comment:
This could just be in `ratOps.ml` to not pollute the top-level module namespace.